src/utils/valid_input_reg_exp.ts

```diff
@@ -0,0 +1,5 @@
+const validInputRegExp = new RegExp('^[0-9]*[.]?[0-9]*$');
+
+export default function isValidInput(input: string): boolean {
```

Comment:
This is a pretty generic choice for a name. It doesn't tell me anything about what is valid. Consider something like `isValidNumericUserInputString`.

Reply:
Updated in b9db4ac